support-frontend/assets/components/serverSideRendered/newspaperArchiveHeader.tsx

```diff
@@ -1,18 +1,175 @@
-import { Global } from '@emotion/react';
+import { css, Global } from '@emotion/react';
+import {
+	palette,
+	space,
+	textSansBold14,
+	until,
+	visuallyHidden,
+} from '@guardian/source/foundations';
+import { Link, SvgGuardianLogo } from '@guardian/source/react-components';
 import { PrerenderGlobalStyles } from 'helpers/rendering/prerenderGlobalStyles';
 import { guardianFonts } from 'stylesheets/emotion/fonts';
 import { reset } from 'stylesheets/emotion/reset';
 
+// The newspapers.com site has a layout change at this breakpoint
+const from991px = '@media (min-width: 991px)';
+const until990px = '@media (max-width: 990px)';
+
+const heading = css`
+	background-color: ${palette.neutral[100]};
+	${textSansBold14};
+	color: #4a4a4a;
+	margin: ${space[2]}px auto;
+	max-width: fit-content;
+	text-align: center;
+`;
+
+const body = css`
+	background-color: #4a4a4a;
+	color: ${palette.neutral[100]};
+	${textSansBold14};
+	display: flex;
+	justify-content: space-between;
+
+	${until.tablet} {
+		padding: 0;
+	}
+
+	padding-top: ${space[4]}px;
+`;
+
+const description = css`
+	${until990px} {
```

Comment:
These are unusual breakpoints compared to our Guardian ones.  Presume specified by newspaper archives themselves?

Reply:
Yep, from Helene "The 991 px is specific to the Newspaper archive website - so if we're able to use it it would be good as the changes of layout will follow what happens on the rest of the page".
So it's not specified by them but it matches their layout